modules/FlaskServer.py

```diff
@@ -52,22 +52,22 @@ def Party():
 
         @server.route("/encounter", methods=["GET"])
         def Encounter():
-            encounter = GetEncounterLog()["encounter_log"].pop()["pokemon_obj"]
-            if encounter:
-                stats = GetStats()
-                response = json.loads("{}")
-                if stats:
-                    try:
-                        encounter["stats"] = stats["pokemon"][encounter["name"]]
+            if len(GetEncounterLog()["encounter_log"]) > 0:
```

Comment:
Should probably just update this so the route doesn't need to call GetEncounterLog() and read the stats file twice, unnecessarily

Reply:
Ah, yes, I'll set `GetEncounterLog()["encounter_log"]` to a variable